### DISCUSSION TITLE
Can´t find the right syntax

### DISCUSSION THREAD

**Discussion by HofeBY** [arendst/Tasmota#18188] [Q&A] [answered]
I want to use variable year to store in year´s Name.
This ist what work´s well if i use in script.

fr=fo("/AMT681-2023.csv" 2)

But what do i have to change that every year will be replaced automatically.

thx.

**Comment by gemu2015** [comment, marked as answer]
you may concatenate a string by "adding"
path = "/AMT681-"+s(0year)+".csv"

**Reply by HofeBY** [reply]
Yesss, it work´s.

Thx a lot gemu2015
